paper/paper.md

```diff
@@ -0,0 +1,287 @@
+---
+title: 'Connectome Mapper 3: A Flexible and Open-Source Pipeline Software for Multiscale Multimodal Human Connectome Mapping'
+tags:
+  - Python
+  - neuroscience
+  - pipeline
+  - workflow
+  - connectome
+  - multi-scale
+  - MRI
+  - BIDS
+  - BIDS App
+  - containerized workflows
+  - nipype
+authors:
+  - name: Sebastien Tourbier
+    orcid: 0000-0002-4441-899X
+    affiliation: 1
+  - name: Joan Rue-Queralt
+    orcid: 0000-0001-5680-4540
+    affiliation: 1
+  - name: Katharina Glomb
+    orcid: 0000-0002-4596-4386
+    affiliation: 1
+  - name: Yasser Aleman-Gomez
+    orcid: 0000-0001-6067-8639
+    affiliation: 1
+  - name: Emeline Mullier
+    orcid: 0000-0001-6209-0791
+    affiliation: 1
+  - name: Alessandra Griffa
+    orcid: 0000-0003-1923-1653
+    affiliation: 2
+  - name: Mikkel Schöttner
+    orcid: 0000-0002-4521-9837 
+    affiliation: 1
+  - name: Jonathan Wirsich
+    orcid: 0000-0003-0588-9710
+    affiliation: 3
+  - name: Anıl Tuncel
+    orcid: 0000-0003-0317-2556
+    affiliation: 4
+  - name: Jakub Jancovic
+    orcid: 0000-0002-3312-3918
+    affiliation: 5
+  - name: Meritxell Bach Cuadra
+    affiliation: 3,6
+    orcid: 0000-0003-2730-4285
+  - name: Patric Hagmann
+    orcid: 0000-0002-2854-6561
+    affiliation: 1
+affiliations:
+ - name: Radiology Department, Centre Hospitalier Universitaire Vaudois and University of Lausanne (CHUV-UNIL), Switzerland
+   index: 1
+ - name: Medical Image Processing Lab (MIP:Lab), Ecole Polytechnique Federale de Lausanne (EPFL), Switzerland
+   index: 2
+ - name: EEG and Epilepsy Unit, University Hospitals and Faculty of Medicine of Geneva, University of Geneva, Geneva, Switzerland
+   index: 3
+ - name: Blue Brain Project, BBP-CORE, Ecole Polytechnique Federale de Lausanne (EPFL), Switzerland
```

Comment:
Hi @sebastientourbier the affiliation is approved. Here's the correct wording below
```
 Blue Brain Project, École polytechnique fédérale de Lausanne (EPFL), Campus Biotech, Geneva, Switzerland
```